_pages/dev/features/reorder.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Reorder
+---
+
+The reorder feature allows you to create cart from an existing order.
+
+## Requirements
```

Comment:
I think reorder only works for b2b, so it should probably be clear from the doc that this is the case.